include/openssl/ocsp.h.in

```diff
@@ -198,7 +198,7 @@ OCSP_REQ_CTX *OCSP_sendreq_new(BIO *io, const char *path, OCSP_REQUEST *req,
                                int maxline);
 int OCSP_sendreq_nbio(OCSP_RESPONSE **presp, OCSP_REQ_CTX *rctx);
 
-/* TODO: remove this (documented but) meanwhile obsolete function? */
+OSSL_DEPRECATEDIN_3_0
```

Comment:
Must be enclosed in `#ifndef OPENSSL_NO_DEPRECATED`

Reply:
Doh!  And that fixes why "make update" doesn't work.  Thanks!  Fixup commit pushed.